gdcdictionary/schemas/treatment.yaml

```diff
@@ -3417,10 +3854,11 @@ properties:
       - Orantinib
       - Oraxol
       - Oregovomab
+      - Orelabrutinib
       - Ormaplatin
       - Ortataxel
       - Orteronel
-      - OSI-774 (erlotinib; Tarceva)
+      - Orvacabtagene Autoleucel
       - Osilodrostat
       - Osimertinib
       - Other
```

Comment:
Do we need to remove "Other" here (in addition to removing it from the deprecated_enum section)?

Reply:
@wteouchicago we are not removing "Other". Instead, we are removing it from the deprecated list, as we need it to stay on the list of values.